spec/support/page_objects/users/show.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Users
+    class AddProviderLink < SitePrism::Section
+      element :link, ".add-provider-to-user"
+    end
+
+    class AddLeadSchoolLink < SitePrism::Section
+      element :link, ".add-lead-school-to-user"
+    end
+
+    class Show < PageObjects::Base
+      set_url "/system-admin/users{/id}"
+      element :add_provider, "a", text: "Add provider"
+      element :add_lead_school, "a", text: "Add lead school"
+      elements :providers, ".provider-row dd a"
+      elements :lead_schools, ".lead-school-row dd a"
+      # sections :add_provider, AddProviderLink, ".provider-row"
```

Comment:
No comment 😆

Reply:
When you say fix the spacing, are we talking about leaving a space between the dropdown and the continue button, does the alignment of anything need to change?

Reply:
ok ive changed the spacing and removed the comments, let me know if there are any other changes needed =)